src/algebra/big_operators/associated.lean

```diff
@@ -0,0 +1,120 @@
+/-
+Copyright (c) 2018 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Jens Wagemaker, Anne Baanen
+-/
+
+import algebra.associated
+import algebra.big_operators.basic
+
+/-!
+# Products of associated, prime, and irreducible elements.
+
+This file contains some theorems relating definitions in `algebra.associated`
+and products of multisets and finsets.
+
+-/
+
+variables {α β γ δ : Type*}
+
+local infix ` ~ᵤ ` : 50 := associated
```

Comment:
```suggestion
-- the same locale notation used in `algebra.associated`
local infix ` ~ᵤ ` : 50 := associated
```

Reply:
Should we turn it into `localized "infix ..."`?

Reply:
I'd even argue for making this a global notation, in another PR.